src/RulesMD/MarkdownCreator.cs

```diff
@@ -0,0 +1,354 @@
+﻿// ------------------------------------------------------------------------------
```

Comment:
Strangely enough, this is passing the license header check in the debug build, but not in the release build. Is it possible to generate this file at build time instead of having it in the repo?

Reply:
Yes, we are now preprocessing the templates before the build and writing the output to the obj folder.

Reply:
Thanks! That's much nicer